### PR TITLE
Make it compliant to GDI

### PR DESCRIPTION
**Pull request by hcvdwerf**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>HealthriCatalogApp</title>
+  <title>gdiCatalogApp</title>
```
Comment (brunopacheco1) [inline src/index.html:5]: What do you think of 'GDI - User Portal'?

File: src/app/app.component.spec.ts [hunk 2]
```diff
@@ -19,6 +19,6 @@ describe('AppComponent', () => {
     const fixture = TestBed.createComponent(AppComponent);
     fixture.detectChanges();
     const compiled = fixture.nativeElement as HTMLElement;
-    expect(compiled.querySelector('.content span')?.textContent).toContain('healthri-catalog-app app is running!');
+    expect(compiled.querySelector('.content span')?.textContent).toContain('gdi-catalogue-app app is running!');
```
Comment (brunopacheco1) [inline src/app/app.component.spec.ts:22]: What do you think of renaming all references to `gdi-userportal`?

**Review by brunopacheco1 — APPROVED** [review]

**Comment by brunopacheco1** [comment]
Can you please add license Apache 2?

File: src/app/app.component.spec.ts (hunk 2 above)
Comment (hcvdwerf) [reply src/app/app.component.spec.ts:22]: I have changed it now to make it consistent. little bit confused. From my understanding the catalogue is part of the user portal. User portal will be a seperate project where all comes together?

File: src/index.html (hunk 1 above)
Comment (hcvdwerf) [reply src/index.html:5]: fixed

**Comment by hcvdwerf** [comment]
> Can you please add license Apache 2?

done

**Pull request merged**